src/main/resources/view/AppointmentListCard.css

```diff
@@ -0,0 +1,6 @@
+#priority {
+    -fx-text-fill: black;
+    -fx-padding: 2;
+    -fx-background-radius: 50;
+    -fx-font-family: "Segoe UI Semibold";
+}
```

Comment:
Use CSS class instead of CSS id, the ID must be unique throughout the entire window, which it evidently is not.

Reply:
thx changed.

Reply:
thx changed